File: Samples/Scripts/Spark/Python/GettingStarted/HiveSelect.py
from pyspark.sql import HiveContext

sqlContext = HiveContext(sc)

#Queries are expressed in HiveQL
sqlContext.sql(("create External table if not exists recommend_Ratings1(criticid string,movieid string,rating double) ROW FORMAT DELIMITED FIELDS TERMINATED BY ',' LOCATION '/Data/Ratings/'"))
output=sqlContext.sql("FROM recommend_Ratings1 SELECT criticid,movieid,rating")

output.show()

File: Samples/Scripts/Spark/Scala/GettingStarted/HiveSelect.scala
// sc is an existing SparkContext.
val spark = new org.apache.spark.sql.hive.HiveContext(sc)

spark.sql("create External table if not exists recommend_Ratings1(criticid string,movieid string,rating double) ROW FORMAT DELIMITED FIELDS TERMINATED BY ',' LOCATION '/Data/Ratings/'")

// Queries are expressed in HiveQL
spark.sql("FROM recommend_Ratings1 SELECT criticid,movieid,rating").collect().foreach(println)